Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:latest
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get update -y && \
+  apt-get install -y --no-install-recommends git g++ cmake ninja-build llvm-15-dev libz-dev libglew-dev flex bison libfl-dev libboost-thread-dev libboost-filesystem-dev nvidia-cuda-toolkit-gcc ca-certificates
```

Comment:
* you can use `apt update` and `apt install` instead of `apt-get`
* When installing stuff with `apt ` in a Dockerfile, you might want to end the command with `rm -rf /var/lib/apt/lists/*`. [Best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
* You might want to remove the cloned gpuocelot folder after building & installing it